packages/framework/src/Model/Transport/TransportData.php

```diff
@@ -93,5 +99,6 @@ public function setFromEntity(Transport $transport, array $transportDomains)
             $domains[] = $transportDomain->getDomainId();
         }
         $this->domains = $domains;
+        $this->payments = $transport->getPayments()->toArray();
```

Comment:
Shouldn't `$transport->getPaymnets()` return the target array instead of `Collection`? The interface would become cleaner